### DISCUSSION TITLE
Can't install any new packages in atom

### DISCUSSION THREAD

**Discussion by Cyberblackstone** [atom/atom#22243] [Q&A] [answered]
Hi! I have recently installed atom and want to install packages on my mac (10.13). I want to install the term3 package for easier access to my terminal instead of having to open it in another window. But after trying to install I get error messages (I have attached the log below). I can't understand what they mean (I am newbie) and would really appreciate it if any one could help me fix this issue. 
Thanks
[2021-04-20T07_10_28_645Z-debug.log](https://github.com/atom/atom/files/6341499/2021-04-20T07_10_28_645Z-debug.log)

**Comment by sadick254** [comment, marked as answer]
@nodiguia The failure you are seeing is as a result of a [native package that term3 utilizes](https://github.com/iiegor/ptyw.js). When installing the term3 package npm fails when trying to rebuild ptyw.js. You could try using other terminals like [terminus](https://atom.io/packages/terminus) or [x-terminal](https://atom.io/packages/x-terminal)

**Reply by Cyberblackstone** [reply]
Thanks!! that made my day !!
